webclient/neidentakce/migrations/0003_auto_20230328_1612.py

```diff
@@ -7,7 +7,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('dokument', '0005_auto_20230328_1538'),
+        #('dokument', '0005_auto_20230328_1538'),
```

Comment:
@pesikj tohle neexistuje, tak jsem to zakomentoval protoze mi padalo nasazeni

Reply:
Ok.